.github/workflows/tag-model-runs.yaml

```diff
@@ -0,0 +1,72 @@
+# Workflow that can be manually dispatched to tag existing model runs with
+# a new run type (e.g. `test`, `baseline`, `candidate`, `final`).
+#
+# Gated such that it's impossible to tag runs older than the current
+# assessment cycle, where each assessment cycle starts in April.
+
+name: tag-model-runs
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_type:
+        type: choice
+        description: New/replacement run type to apply to specified runs
+        options:
+        - junk
+        - rejected
+        - test
+        - baseline
+        - candidate
+        - final
+        default: test
+        required: true
+      run-ids:
```

Comment:
**[Nitpick, non-blocking]** So far we've been using snake_case for workflow input variables in this repo; I don't feel strongly about this vs. kebab-case but we should probably stay consistent.

Reply:
Fixed in [da1c41d](https://github.com/ccao-data/model-res-avm/pull/183/commits/da1c41d7f86756325f589a73ab2f35727192544a)!